roseau/load_flow_single/models/buses.py

```diff
@@ -208,6 +210,30 @@ def max_voltage(self) -> Q_[float] | None:
             else Q_(self._max_voltage_level * self._nominal_voltage, "V")
         )
 
+    @property
+    def short_circuit(self) -> bool:
+        """Return the list of short-circuits of this bus."""
+        return self._short_circuit
+
+    def add_short_circuit(self) -> None:
+        """Add a short-circuit by connecting all the phases together with a ground."""
+        from roseau.load_flow_single import PowerLoad
+
+        for element in self._connected_elements:
+            if isinstance(element, PowerLoad):
```

Comment:
Last time we discussed this we said that current loads are also not allowed here right?

Reply:
Oh yes I've forgotten that, it's because we multiply by `u / std::abs(u)` which is `nan` in case of short circuit